app/assets/stylesheets/responsive.scss

```diff
@@ -1,6 +1,7 @@
 /* Bootstrap overrides go here. This is included after bootstrap. */
 $navCollapseBackgroundColor: #f7f7f7;
 $navCollapseColor: #777;
+$navbarActiveBorderColor: #aaa;
```

Comment:
Ok. We'll want to be sure to double check what happens in the schools because they override this variable (they do it in `nucore-overrides.scss`). It might be worth testing out here in open by setting something in that file and make sure it applies where it's supposed to and not where it's not.